gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Tue Apr 06 10:19:50 MSK 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
the latest is 8.8

Reply:
@terrakok All updated, please check again when you have time. Thank you for your help!